craft_providers/multipass/multipass_instance.py

```diff
@@ -50,8 +51,13 @@ def _rootify_multipass_command(
     """
     sudo_cmd = ["sudo", "-H", "--"]
 
-    if env is not None:
-        sudo_cmd += env_cmd.formulate_command(env)
+    if cwd is not None:
+        cwd_path: Optional[pathlib.Path] = pathlib.Path(cwd)
```

Comment:
I don't understand the usage of `Optional` here...

Reply:
reworked anyways to avoid this